docs-new/docs/installation/configuration/cert-manager.md

```diff
@@ -61,5 +62,23 @@ Note the following about these fields:
   must be `keptn-certs`.
 * Substitute the namespace placeholders with your namespace, where Keptn is installed.
 
-See the [CA Injector](https://cert-manager.io/docs/concepts/ca-injector/)
-documentation for more details.
+## Injecting CA Annotations
+
+`cert-manager.io` supports specific annotations for
+injectable resources depending on the injection source.
+To configure these annotations, modify the `global.caInjectionAnnotation` Helm value.
+See the [CA Injector](https://cert-manager.io/docs/concepts/ca-injector/) documentation for more details.
+
+Here is an example `values.yaml` file demonstrating the configuration of CA injection
+by using the `cert-manager.io/inject-ca-from` annotation:
+
+```yaml
+global:
+  certManagerEnabled: false # disable Keptn Cert Manager
+  caInjectionAnnotations:
+    cert-manager.io/inject-ca-from: keptn-system/keptn-certs
+```
+
+Refer to the
+[Customizing the configuration of components](https://keptn.sh/latest/docs/installation/#customizing-the-configuration-of-components)
```

Comment:
please use a relative link here instead of an absolute one
```suggestion
[Customizing the configuration of components](../index.md#customizing-the-configuration-of-components)
```